docs/network-flow-visibility.md

```diff
@@ -352,6 +358,22 @@ commands:
 ./infra/vagrant/push_antrea.sh --flow-collector <externalFlowCollectorAddress>
 ```
 
+## IPFIX Collector
```

Comment:
We can have a section called Flow Collectors and put IPFIX Collector and ELK Flow Collector as sub-sections